biit_server/database.py

```diff
@@ -65,3 +65,36 @@ def query(self, field, operation, value) -> List[Dict[str, Any]]:
             return [value for value in results]
         except Exception:
             return False
+
+    def update(self, id, update_dict) -> bool:
+        """Helper function to query documents based on parameters.
+
+        Args:
+            id (str, int): The id of the document you want (email for us).
+            update_dict (Dict[str, Any]): A dictionary of the fields you want to update
```

Comment:
does this need to be full (example if i only pass firstname will it update properly?)